MAUI/ImageEditor/toolbar-customization.md

```diff
@@ -229,6 +235,8 @@ Change the default colors of the `ColorPalette` in the toolbar.
 
 N> Color palette colors are common for Text, Shape and Pen.
 
+![Toolbar Color Palate in .NET Maui ImageEditor](images/toolbar/imageeditor-colorpalate.png)
```

Comment:
Palate => spell error

Palette, change this in the image name too

Reply:
Changed